sw/airborne/Makefile

```diff
@@ -38,16 +38,16 @@ VPATH = .
 ifneq ($(MAKECMDGOALS),clean)
   include $(AIRCRAFT_BUILD_DIR)/Makefile.ac
   $(TARGET).srcs += $($(TARGET).EXTRA_SRCS)
-  include ../../conf/Makefile.local
+  include $(PAPARAZZI_HOME)/conf/Makefile.local
```

Comment:
Most of this file is actually useless, but you're probably right, it could be somewhere else in SRC part. Same for most of the makefiles in `conf` I guess.